### PR TITLE
[RFR] Deployment process

### PR DESCRIPTION
**Pull request by Kmaschta**
- [x] Prod & staging server deployment
- [x] Prod & staging static deployment
- [x] Documentation

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+@task
+def setup_api():
+    print(green('Installing dependencies ...'))
+    sudo('apt --yes update && apt --yes upgrade')
+    sudo('curl -sL https://deb.nodesource.com/setup_5.x | bash -')
+    sudo('apt --yes install build-essential')
+    sudo('apt --yes install libkrb5-dev')
+    sudo('apt --yes install nodejs')
+    sudo('apt --yes install supervisor')
```
Comment (jpetitcolas) [inline fabfile.py]: Do we still need supervisor with PM2?

**Comment by jpetitcolas** [comment]
:+1:

File: fabfile.py (hunk 1 above)
Comment (Kmaschta) [inline fabfile.py]: PM2 is only for dev

**Comment by djhi** [comment]
:+1:

**Pull request merged**